templates/partial/_tab-actions.html

```diff
@@ -3,9 +3,9 @@
     <hr class="p-separator" />
     <h2>Actions</h2>
   </noscript>
-  <aside class="p-accordion" role="tablist" aria-multiselect="true">
+  <aside class="p-accordion" role="tablist" aria-multiselect="true" data-js="actions">
```

Comment:
do we want to add the expanded accordion to the url or it is to much?

Reply:
The issue with that is you can have multiple open at once, so not sure how we would codify that into the URL without it being hideous